index.bs

```diff
@@ -1048,6 +1048,8 @@ as well as obtaining and changing the size of, a single file.
 A {{FileSystemSyncAccessHandle}} offers synchronous methods. This allows for higher performance on
 contexts where asynchronous operations come with high overhead, e.g., WebAssembly.
 
+A {{FileSystemWritableFileStream}} has a <dfn for="FileSystemWritableFileStream">file position cursor</dfn> initialized at byte offset 0 from the top of the file.
```

Comment:
Both a `FileSystemSyncAccessHandle` and `FileSystemWritableFileStream` should have a cursor, right? And does this need to be explicitly initialized to 0 in the "create a new FileSystemSyncAccessHandle" algorithm below?

Reply:
It's usually fine to state the default value of an internal field upfront like is done here.